eventlogger/hapi_server_v17.js

```diff
@@ -0,0 +1,78 @@
+var assign = require('lodash').assign;
+
+module.exports.watch = function (logger, server, options) {
+  options = assign({}, {
+    ignorePaths: []
+  }, options);
+
+  const ignorePaths = options.ignorePaths;
+
+  function createLogEntry(request, log_type) {
+    const took = Date.now() - request.info.received;
+
+    return {
+      log_type,
+      took,
+      req: request,
+      res: request.response
+    };
+  }
+
+  async function onRequest(request, h) {
+    if (ignorePaths.indexOf(request.path) >= 0) {
+      return h.continue;
+    }
+    logger.info({ log_type: 'request', req: request });
+    return h.continue;
+  }
+
+  function onRequestError(request, error) {
+    logger.error({
+      log_type: 'request_error',
+      req: request,
+      res: request.response,
+      err: error,
+      payload: request.pre &&
+      request.pre._originalPayload ||
+      request.payload
```

Comment:
I know it was already like that but, is it safe to log the payload like this? what if there were passwords or things like those?